src/main/java/org/mvplugins/multiverse/core/commands/WhoCommand.java

```diff
@@ -101,6 +103,7 @@ void onWhoAllCommand(
     void onWhoCommand(
             MVCommandIssuer issuer,
 
+            @Optional
             @Syntax("<world>")
             @Description("{@@mv-core.who.world.description}")
             LoadedMultiverseWorld inputtedWorld,
```

Comment:
Can just add a @Flag("resolve=issuerAware") to get the world from player itself

Reply:
Oh, I always forget that...